fabtests/test_configs/shm/shm.exclude

```diff
@@ -7,7 +7,6 @@ inject_test
 -e dgram
 
 # Exclude tests that use sread/polling
--S
```

Comment:
I am not sure if any test is using `-S` for sread. This may need change if that is true